apps/explorer/src/pages/home/Home.tsx

```diff
@@ -93,10 +93,10 @@ function Home() {
                     </TabPanels>
                 </TabGroup>
             </div>
-            {/* TODO: Add the popular packages component here :) */}
-            {/* <div style={{ gridArea: 'packages' }} className="mt-5 bg-gray-60">
-                Popular packages
-            </div> */}
+
+            <div style={{ gridArea: 'packages' }}>
+                <TopPackagesCard />
```

Comment:
one callout here:

I was having issues nesting two TabGroups (one for the Packages table header, and the other for the date filter) as it was inheriting styling from the parent tab group and marking everything in the date filter as selected. 

so in order to not have to hoist the filter state up to the home page I dropped the components down a level into `TopPackagesCard`

Reply:
Yeah tbh datefilter being tabs is really wrong, technically. It should be a radio button. That's my bad.

Reply:
Created a JIRA to cover the follow up work